package com.petproject.workflow.data.network.exceptions

class AuthException: RuntimeException()